scripts/manifests/chromemanifest.json

```diff
@@ -31,10 +31,10 @@
   // testing version allows loading unit test libraries from CDNs
   "content_security_policy": "default-src 'self'; img-src data:; style-src 'self' https://unpkg.com; script-src 'self' https://unpkg.com",
   "icons": {
-    "16": "icons/icon-small-colored.svg",
-    "32": "icons/icon-small-colored.svg",
-    "48": "icons/icon-large.svg",
-    "96": "icons/icon-large.svg"
+    "16": "icons/icon.svg",
+    "32": "icons/icon.svg",
+    "48": "icons/icon.svg",
+    "96": "icons/icon.svg"
```

Comment:
Chrome does [not support SVG icons](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/icons#browser_compatibility).

Reply:
:roll_eyes:

Reply:
But well… I have defined it the same in all other add-ons, so this would need to be adjusted.

Could we at least define a large size as SVG or so (say 128px), so Firefox can properly use SVGs… and define some static legacy PNGs for Chrome/ium browsers?

Reply:
Okay tested it and Chromium (at least) seems to ignore the SVg icon then, so let's include it in case Chrome/ium gets support for that feature in the future.

Reply:
> But well… I have defined it the same in all other add-ons, so this would need to be adjusted.

As far as I am aware, your other add-ons do not currently support Chrome. It also will not be an issue with the Awesome Emoji-Picker, since it only uses PNG icons.